### ISSUE TITLE
CC0017 Change to auto property codefix removes multiple variable declaration.

### ISSUE THREAD

**Issue by thorgeirk11** [code-cracker/code-cracker#512]
While I was working on #500 I encountered another issue:
CodeFix removes multiple variable declaration.
---
The following:
```CSharp
public int x, y;
public int X
{
    get { return x; }
    set { x = value; }
}
```
Get transformed into
```CSharp
public int X { get; set; }
```
The `y` gets removed.

**Comment by thorgeirk11** [comment]
I will fix this along with #500 :+1:

**Comment by giggio** [comment]
Ok, go for it!

**Comment by giggio** [comment]
Closed on #515
